File: python/6_kyu/count_characters.py
# https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/python

def count(string):
    output = {}
    for c in string:
        output[c] = output[c] + 1 if c in output else 1
    return output


print(count("aba"))

File: CodeWars/javascript/6_kyu/count_characters.js
// https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/javascript

function count(string) {
  const output = {};
  for (let c of string) output[c] = output[c] ? output[c] + 1 : 1;
  return output;
}

console.log(count("aba"));
